### ISSUE TITLE
Feature Request : Confirmation #

### ISSUE THREAD

**Issue by chazdiezal** [alextselegidis/easyappointments#48]
Please implement a generated booking confirmation # when a customer successfully books an appointment.  This conf # should be included in the customer's successful email and stored in a table for view on the calendar backend.

**Comment by alextselegidis** [comment]
You mean that an appointment needs to be valid only if customers confirm them from their email notifications?

**Comment by chazdiezal** [comment]
Yes, I think this could be incorporated as well.  Some sort of validation is needed to serve as to validate if the person scheduling the appointment is in fact the email owner.  If we require the customer to select a "verify appointment" button in the email and somehow write back to the backend that the appointment was confirmed.  Also I would like to see a six to eight digit confirmation # assigned to the booking.  For example, confirmation #12345678, would be included in the customer's system generated email reply after the customer verifies the appointment.

**Comment by alextselegidis** [comment]
Ok, yeah it sounds pretty reasonable. I will leave it for a future release through. I plan to create a new setting for this in the backend/settings page where admin users can set if they need an email confirmation in order to set an appointment as valid.

**Comment by netsparkle** [comment]
Is there any further development of a confirmation step?

**Comment by alextselegidis** [comment]
This ticket has being pushed back in favor of others.

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="https://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>
